engine/src/main/java/org/terasology/physics/shapes/ConvexHullShape.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.physics.shapes;
+
+/**
+ * Represents a convex hull collision entity in the physics engine.
```

Comment:
I'd remove "in the physics engine". As it's part of the "physics" package, I think it's clear enough. Other shapes' javadoc have the same problem.

Reply:
See comment above for similar issue.